components/kserve/README.md

```diff
@@ -4,7 +4,7 @@ Organization: KServe
 
 Organization Description: KServe is a highly scalable and standards based Model Inference Platform on Kubernetes for Trusted AI
 
-Version information: 0.7.0
```

Comment:
Maybe in the readme, we should describe which commit tag is the KServe 0.7.0 version so Kubeflow 1.5 users can point to the correct component.

Reply:
Maybe add something like

**Note**: To use the 0.7.0 version of this component which runs on Kubeflow 1.5, then change the `load_component_from_url` in the usage section with the following YAML instead:

```yaml
https://raw.githubusercontent.com/kubeflow/pipelines/b1c803bfc3ae5ec4a90fbf228a54c5f8f0cf5b5f/components/kserve/component.yaml
```

Reply:
Sound good! I think we could use the latest stable tag `v1.8.1` instead of the commit hash to make it look even better.